src/Aspire.Hosting.Analyzers/AppHostAnalyzer.cs

```diff
@@ -19,6 +19,11 @@ public partial class AppHostAnalyzer : DiagnosticAnalyzer
 
     public override void Initialize(AnalysisContext context)
     {
+        if (context is null)
```

Comment:
Analyzers aren't "public API" - we don't ship them in a library where someone can reference them. This `Initialize` method is called by Roslyn and it won't ever pass in a null context.

So we can revert this change, and the corresponding test in `AnalyzersPublicApiTests.cs`.